rpcd/playbooks/roles/rpc_support/tasks/configure_unattended_upgrades.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2014, Rackspace US, Inc.
```

Comment:
desperately needs updating, you should update this every time you update a file with it in it to be honest (not that we are good about it).